README.md

```diff
@@ -142,6 +156,12 @@ None right now.
 - **How to sign up**: First you have submit two PRs then let us know you’ve completed the challenge by filling out the form on our [official Hacktoberfest page](https://camunda.com/hacktoberfest2020/) and we will send you a e-mail. The repos are: [Camunda BPM](https://github.com/camunda/), [bpmn.io](https://github.com/bpmn-io/), and [Zeebe-io](https://github.com/zeebe-io/).
 - **Notes**: Check out Camunda [official Hacktoberfest page](https://camunda.com/hacktoberfest2020/) for more info & how to get the T-shirt.
 
+#### **Circle-CI** (T-shirt, stickers, limited edition orb magic pin)
+
+- **Requirements**: Create 4 PRs on any of [Circle-CI Repos](https://github.com/circleci) as well as make sure you fill the form [here](https://circleci-community.typeform.com/to/mErGL0nv) after making them.
+- **Swag**: Sticker pack and t-shirt
```

Comment:
Please let the great people know how they can earn the Orb pin too! 😄

Reply:
Or I have to mention exact what they need to do for challenge-2 ?